filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1581,6 +1581,9 @@ bool OpenGLDriver::isWorkaroundNeeded(Workaround workaround) {
             return mContext.bugs.split_easu;
         case Workaround::ALLOW_READ_ONLY_ANCILLARY_FEEDBACK_LOOP:
             return mContext.bugs.allow_read_only_ancillary_feedback_loop;
+        case Workaround::ADRENO_UNIFORM_ARRAY_CRASH:
+            return (strstr(
+            mContext.getRenderer(), "Adreno") != NULL);
```

Comment:
please implement this like the ones above, by creating a field in `mContext.bugs` and initializing it during construction.